fedimint-startos/manifest.yaml

```diff
@@ -1,11 +1,11 @@
 id: fedimintd
 title: "Fedimint"
-version: 0.8.1
+version: 0.8.1.1
```

Comment:
StartOS uses https://github.com/Start9Labs/emver-rs/tree/master to handle bumping package versions when the underlying software version hasn't changed.

Ex:
Mempool bump `3.0.0.3`: https://github.com/Start9Labs/mempool-startos/blob/v3.0.0.3/manifest.yaml#L3

In the UI it looks like
<img width="324" height="127" alt="Screenshot_20250919_001834" src="https://github.com/user-attachments/assets/191adadb-d102-41c3-819a-18bcd13f5914" />